---
title: "Taxonomic Athentication" 
author: "Hyunmin Kim"
last updated : "`r Sys.Date()`"
output: md_document
knitr:
  opts_chunk:
    echo: false
    warning: false
    message: false
---

## Trimmed and Pair-Merged Read Lengths 
- Reads were merged using LeeHom

```{r len, echo=FALSE, warning=FALSE, message=FALSE}

md_save_pdf <- function(p, f, width = 10, height = 20) {
  print(p)
  ggsave(filename = f, plot = p, width = width, height = height)
  cat(sprintf("[PDF](%s)\n", f))
}   


library(data.table)
library(ggplot2)
library(dplyr)
tt <- fread("len.txt", sep="\t", header=FALSE)
setnames(tt,c("group","sample","read_len","count"))
d<- tt[, .(
  total_reads = sum(count),
  short_reads = sum(count[read_len < 20]),
  perc_short = round(sum(count[read_len < 20]) * 100 / sum(count),digits=4)
), by = sample]
fwrite(d,"../../bigdata/results/short_perc.tsv",sep="\t") 

#tt = fread("n.txt")
#setnames(tt,c("group","sample","leehom_mated","leehom_failed"))
#tt[ , fail_perc := round(leehom_failed / (leehom_mated + leehom_failed) * 100,digits=4 )];
#fwrite(tt,"../../bigdata/results/fail_perc.tsv",sep="\t") 


tt$sample <- factor(tt$sample , levels = unique(tt$sample))
tt[, type := sub("^[^_]+_(.*)$", "\\1", sample)]
tt[, type := ifelse(type=="ExtrBlank","Blank",type)]


ggplot(tt, aes(x = read_len, y = count, color = type)) +
  geom_line(alpha = 1, linewidth=1.5) +
  facet_wrap(~group, scales = "free") +   
  labs(x = "read_len", y = "count", title = "Distribution of Read_len") +
  theme_minimal() +
  theme(legend.position = "bottom")


```

## BWA Best Scores 

![bwa_score](figs/bwa_score_grid.png)

## Kraken2 Contamination Report 

| Taxa           | Included | Description                                      |
|----------------|:--------:|-------------------------------------------------|
| Archaea        | ✅       | Single-celled microorganisms distinct from bacteria |
| Bacteria       | ✅       | Prokaryotic microorganisms                       |
| Human          | ✅       | Host genome sequences                            |
| Plasmid        | ✅       | Extrachromosomal DNA elements                    |
| Viral          | ✅       | Viruses including phages                         |
| Fungi          | ✅       | Eukaryotic organisms such as yeasts and molds   |
| Protozoa       | ✅       | Single-celled eukaryotes                         |
| UniVec         | ✅       | Vector contamination sequences                    |
| UniVec_Core    | ✅       | Core vector sequences                             |

```{r contam, echo=FALSE, warning=FALSE, message=FALSE}
library(knitr)
tt=fread("kr_report_domain2phylom.csv")
tt[,classified := 100 - unclassified]
tt[,unclassified := NULL]
kable(tt, format = "markdown", digits = 2, caption = "Kraken report: domain proportions by sample")

taxa_cols <- setdiff(names(tt), "sample")
depth <- nchar(gsub("[^_]", "", taxa_cols))  # count underscores
taxa_depth <- data.table(taxa = taxa_cols, depth = depth)
tt_long <- melt(tt, id.vars = "sample", variable.name = "taxa", value.name = "percent")
tt_long[, taxa_clean := gsub("_+", "", taxa)]  # remove underscores
tt_long[, depth := nchar(gsub("[^_]", "", taxa))]  # underscore count

tt_long[, rank := fifelse(depth == 0, "classified",
                   fifelse(depth <= 2, "domain",
                   fifelse(depth <= 4, "phylum/clade",
                   fifelse(depth <= 6, "class",
                   fifelse(depth <= 8, "order",
                   fifelse(depth <= 10, "family", "genus/species"))))))]
rank_levels <- c("classified", "domain", "phylum/clade", "class", "order", "family", "genus/species")

tt_long[, rank := factor(rank, levels = rank_levels)]


ggplot(tt_long, aes(x = sample, y = percent, fill = taxa_clean)) +
  geom_bar(stat = "identity", color = "black") +
  facet_wrap(~rank, scales = "free_y") +  # separate panels by depth
  labs(x = "Sample", y = "Percent", title = "Taxonomic Proportions by Kraken Report") +
  theme_minimal() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1),
        legend.position = "bottom") +
  scale_fill_brewer(palette = "Set3")

```

